Mob 47 content liken

jeetjeweet opened the pull request.

jeetjeweet committed:
Like functionaliteit

De like knop liked nu een solution op de homepagina, de solution wordt helemaal meegegeven en alleen userid 1 wordt nu erbij gezet als like.

jeetjeweet committed:
Update SolutionsList.vue

Bookmark knop verplaatst naar onder image. Nu wordt de solution geopent wanneer je op de image drukt.

jeetjeweet committed:
Update Profile.vue

like functionaliteit ook bij profile toegevoegd.

jeetjeweet committed:
Like solution knop

Wanneer je een solution geliked hebt wordt de knop groen. Er wordt alleen nog maar gekeken naar userid 1. Ik heb de inhoud van solutionlist in solutionlistitem gezet zodat het nu in een apart component staat.

jeetjeweet committed:
Update Home.vue

console log delete

Tjerk132 commented on db.json:
```diff
@@ -15,6 +15,11 @@
       "viewCount": 234,
       "coverImage": "https://cdn.vuetifyjs.com/images/cards/cooking.png",
+      "likes": [
+        1,
```
createsolution: likedByUser aangeven,

Tjerk132 commented on src/components/SolutionsList.vue:
```diff
@@ -97,7 +48,24 @@ export default {
+    checkLikes(solution) {
+      let userid = 1;
+      let color;
+      solution.likes.forEach(like => {
```
wordt solution.isLiked boolean

jeetjeweet committed:
Likes list verandert naar boolean isLiked

List is nu een boolean.
De call is verandert naar een patch.
Wanneer een solution is geliket wordt ie ge unliket en wanneer die niet is geliket wordt ie geliket.

Commit:
~UPDATE liked btnColor to inline

Tjerk132 commented on src/views/Profile.vue:
```diff
@@ -120,6 +121,17 @@ export default {
+      }
+      this.solutionContext
+        .likeSolution(solution)
+        .then(console.log('iets doen yeah confirmationus'));
```
Bietje mwa

Tjerk132 committed:
Merge branch 'develop' into MOB-47-Content-Liken
